lib/inference/open_ai_image_generator.rb

```diff
@@ -45,6 +286,168 @@ def self.perform!(prompt, model: "dall-e-3", size: "1024x1024", api_key: nil, ap
               raise "OpenAI API returned #{response.code} #{response.body}"
             else
               json = JSON.parse(response.body, symbolize_names: true)
+              # Add original prompt to response to preserve it
+              json[:original_prompt] = original_prompt
+            end
+          end
+          json
+        end
+      end
+
+      def self.perform_edit_api_call!(
+        images,
+        prompt,
+        mask: nil,
+        model: "gpt-image-1",
+        size: "auto",
+        api_key:,
+        api_url:,
+        n: 1,
```

Comment:
I wonder if there is a more descriptive name we can use for this kwarg. Not sure what `n` stands for here without having to read through method.

Reply:
yeah its tricky, I map to what open ai call the param, we can remap internally, but generally we keep it at 1 anyway so maybe we can even remove for now.